content/working_with/console/widgets/filters.md

```diff
@@ -0,0 +1,52 @@
+
+---
+layout: bt_wiki
+title: Filters
+category: Widgets
+draft: false
+---
+
+Displays the list of defined filters.
+The table has 5 columns:
+
+* **Filter name** - filter ID
+* **Creator** - filter creator
+* **Created** - filter creation date and time
+* **System** - system filter indicator - checked when given filter is a system filter, unchecked otherwise
+* Actions column
+
+![filters]( /images/ui/widgets/filters.png )
+
+The actions column contains the following action icons:
+
+* **Clone filter** opens the **filter clone** modal allowing to create a modified copy of the selected filter (see [defining filter rules]({{< relref "#defining-filter-rules" >}}) for details on rule definition)
+* **Edit filter** available only to `user` filters, opens the **filter rules edit** modal (see [defining filter rules]({{< relref "#defining-filter-rules" >}}) for details on rule definition)
+* **Delete filter** available only to `user` filters, removes the selected filter (see note below)
```

Comment:
```suggestion
* **Delete filter** available only to non-`system` filters, removes the selected filter (see note below)
```

Reply:
we dont mention "users filter above. we do mention system filters so changing the phrasing to simplify the understanding.